extensions/SampleExtension/SettingsProvider/SettingsProvider.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Windows.DevHome.SDK;
+using SampleExtension.Helpers;
+using Serilog;
+
+namespace SampleExtension.Providers;
+
+public class SettingsProvider() : ISettingsProvider
+{
+    private static readonly Lazy<ILogger> _logger = new(() => Serilog.Log.ForContext("SourceContext", nameof(SettingsProvider)));
+
+    private static ILogger Log => _logger.Value;
+
+    string ISettingsProvider.DisplayName => Resources.GetResource(@"SettingsProviderDisplayName", Log);
+
+    public AdaptiveCardSessionResult GetSettingsAdaptiveCardSession()
+    {
+        Log.Information($"GetSettingsAdaptiveCardSession");
+        return new AdaptiveCardSessionResult(new SettingsUIController());
+    }
+
+    public void Dispose()
```

Comment:
For my education: does Dispose() do anything when the class isn't an IDisposable?

Reply:
ISettingsProvider in the Dev Home SDK is IDisposable, so this is in-fact an IDisposable via inheritance.

Reply:
I see, I didn't read the correct doc for IClosable so I didn't understand that for .Net it becomes IDisposable.